config-windows.md

```diff
@@ -82,6 +82,12 @@ The following parameters can be specified (mutually exclusive):
 * **`count`** *(uint64, OPTIONAL)* - specifies the number of CPUs available to the container. It represents the fraction of the configured processor `count` in a container in relation to the processors available in the host. The fraction ultimately determines the portion of processor cycles that the threads in a container can use during each scheduling interval, as the number of cycles per 10,000 cycles.
 * **`shares`** *(uint16, OPTIONAL)* - limits the share of processor time given to the container relative to other workloads on the processor. The processor `shares` (`weight` at the platform level) is a value between 0 and 10,000.
 * **`maximum`** *(uint16, OPTIONAL)* - determines the portion of processor cycles that the threads in a container can use during each scheduling interval, as the number of cycles per 10,000 cycles. Set processor `maximum` to a percentage times 100.
+* **`affinityCPUs`** *(array of objects, OPTIONAL)* - specifies the set of CPU to affinitize for this container.
```

Comment:
nit: why not just `affinity` (since this is a part of cpu struct/group, and other (ajacent) fields do not have CPU in them)

Reply:
Answering to myself.

I see that earlier version had `affinityCPUs` and `AffinityPreferredNumaNodes`, but the latter was dropped.

Still, `affinity` would work, and, if we're to add NUMA affinity later, a name like `NUMAaffinity` (or `affinityNUMA`) would still work.

Reply:
The sub fields could also be `mask` and `group` accordingly 👀

Reply:
Indeed. The less tautology the better.

Reply:
@tianon  @kolyshkin  keeping it similar to how we've defined this in k/k proto file. There was a similar comment in that PR : https://github.com/kubernetes/kubernetes/pull/124285#discussion_r1648429800 

group alone is a keyword in .proto so we couldn't just have mask or group. Thought uniformity would be good across the fields- prefix "cpu" in both fields or neither.

Reply:
similar rationale to why affinityCPUs was used : 
![image](https://github.com/user-attachments/assets/9db9d957-46f6-42ff-a648-2e5c16c7d469)


It is part of WindowsContainerResources struct which can mean cpus/memory or any other resource. So we named it affinityCPUs there. I understand that this struct in runtimespec is specifically for CPU alone. So if you would prefer me to drop "cpu" and just use "affinity" let me know, I can have that updated. Thanks.

Reply:
I imagine there's got to be a way to convince Protobuf to use a keyword as an identifier, right?  Some way to escape it?

Regardless, _this_ spec is for a canonically-JSON-format, which doesn't have any constraints like that (or reserved keys at all), so I definitely still agree with @kolyshkin that we should ditch the duplication in the names and go for the simpler and more consistent all-lowercase versions (`affinity` or `affinities`, `group`, and `mask`).

Reply:
@tianon  have addressed this. Please take a look and let me know if it looks good. Thanks.